sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md

```diff
@@ -51,6 +51,11 @@
 - Support creating `GrantedAuthority` by "roles" claim from id-token for web application.
 
 ## 3.2.0 (2021-03-03)
+### Breaking Changes
+- For the required scopes in auth code flow, use `Directory.Read.All` instead of `Directory.AccessAsUser.All`. ([#18901](https://github.com/Azure/azure-sdk-for-java/pull/18901))
+
+    Now The requested scopes are: `openid`, `profile`, `offline_access`, `User.Read`, `Directory.Read.All`. You can Refer to `AADWebAppConfiguration.java` for detailed information.
```

Comment:
Is it necessary to modify the old release notes for this change? And we plan to modify it, then we need to modify that of azure-spring-boot and azure-spring-boot-starter as well.
How about add GitHub code url with that associate commit id to AADWebAppConfiguration.java?

Reply:
> Is it necessary to modify the old release notes for this change? 

Yes, we already have customer complain about this: https://github.com/Azure/azure-sdk-for-java/issues/22677#issuecomment-872886005

> And we plan to modify it, then we need to modify that of azure-spring-boot and azure-spring-boot-starter as well.

OK, I'll do that. But if I'm the customer, I'll not read changelog of azure-spring-boot and azure-spring-boot-starter, only read azure-spring-boot-starter-active-directory.

> How about add GitHub code url with that associate commit id to AADWebAppConfiguration.java?

Good point.